glamox/src/LandingPage/FrontPage.jsx

```diff
@@ -1,469 +0,0 @@
-import React from "react";
-import styles from "./FrontPage.module.css";
-import Ncarousel from "./Carousel";
-import { Heading } from "@chakra-ui/react";
-import { useNavigate } from "react-router";
-const FrontPage = () => {
-const navigate=useNavigate()
-
-  const handleClick=()=>{
-navigate("/prodlist")
-  }
-  return (
-    <div className={styles.front_page_container}>
-      <div className={styles.caurosel_part}>
-        <Ncarousel />
-      </div>
-      {/* Biggest Payday Discount================================= */}
-      <br />
-      <br />
-      <br />
```

Comment:
Instead of adding 3 <br> tags, can we give margin-bottom or padding-bottom css property?